eco/tests/java/org/apache/heron/eco/parser/EcoParserTest.java

```diff
@@ -33,17 +140,17 @@
       + "# spout definitions\n"
       + "spouts:\n"
       + "  - id: \"spout-1\"\n"
-      + "    className: \"org.apache.heron.sample.TestWordSpout\"\n"
+      + "    className: \"com.twitter.heron.sample.TestWordSpout\"\n"
```

Comment:
I guess these package names should be reverted?

Reply:
@nwangtw 
oh yeah..  Will fix.